api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -883,11 +883,20 @@ message HttpFilter {
   oneof config_type {
     // Filter specific configuration which depends on the filter being instantiated. See the supported
     // filters for further documentation.
+    //
+    // To support configuring a :ref:`match tree <arch_overview_matching_api>`, use an
+    // :ref:`ExtensionWithMatcher <envoy_api_msg_extensions.common.matching.v3.ExtensionWithMatcher>`
+    // with the desired HTTP filter.
```

Comment:
Can you wire up the new `[#extension-category]` stuff here? See other recent examples. I'm surprised this wasn't already done. cc @phlax

Reply:
I see this annotation is applied one layer higher. I would potentially move it here but up to you.

Reply:
im not following i dont see the extension-category

Reply:
There's `  // [#extension-category: envoy.filters.http]` on L304 on the HttpFilter field declaration